storybook/src/css/Button.css

```diff
@@ -0,0 +1,33 @@
+.custom-button {
+    border: none;
+    padding: 12px 20px;
+    color: white;
+    text-align: center;
+    text-decoration: none !important;
+    cursor: pointer;
+    display: inline-block;
+    position: relative;
```

Comment:
Is there a benefit of using relative here? What's the difference between static, relative and absolute?